### ISSUE TITLE
E0080 needs better wording

### ISSUE THREAD

**Issue by jonathandturner** [rust-lang/rust#35223]
From: src/test/compile-fail/E0080.rs

Error E0080 has been updated to the new format, but its wording is awkward.  It can be updated from:

```
error[E0080]: constant evaluation error
  --> src/test/compile-fail/E0080.rs:12:9
   |
12 |     X = (1 << 500), //~ ERROR E0080
   |         ^^^^^^^^^^ attempted to shift left with overflow

error[E0080]: constant evaluation error
  --> src/test/compile-fail/E0080.rs:13:9
   |
13 |     Y = (1 / 0) //~ ERROR E0080
   |         ^^^^^^^ attempted to divide by zero
```

To:

```
error[E0080]: constant evaluation error
  --> src/test/compile-fail/E0080.rs:12:9
   |
12 |     X = (1 << 500), //~ ERROR E0080
   |         ^^^^^^^^^^ attempt to shift left with overflow

error[E0080]: constant evaluation error
  --> src/test/compile-fail/E0080.rs:13:9
   |
13 |     Y = (1 / 0) //~ ERROR E0080
   |         ^^^^^^^ attempt to divide by zero
```

**Comment by shantanuraj** [comment]
Created a pull request https://github.com/rust-lang/rust/pull/35283

**Comment by KiChjang** [comment]
Should be fixed by #35283.
